AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2076,11 +2080,21 @@ - (void)recursivelyClearFetchedData
   });
 }
 
-- (void)visibilityDidChange:(BOOL)isVisible
+- (void)visibileStateDidChange:(BOOL)isVisible
```

Comment:
We'll need a copy of this method so the base class can always respond to the old name.